### PR TITLE
Added Java Coding Program 8

### PR DESCRIPTION
**Pull request by tejashrighogare**

### CODE REVIEW

File: lambdaInArrayList/LambdaInArraylistProgram.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package lambdaInArrayList;
+
+import java.util.*;
+
+public class LambdaInArraylistProgram {
+    //Java ArrayList Program to remove elements of Arraylist using lambda function.
+    public static void main(String[] args) {
+        ArrayList<String> arrayListObject = new ArrayList<String>();
+        arrayListObject.add("Anil");
```
Comment (amitsahu8) [inline lambdaInArrayList/LambdaInArraylistProgram.java]: Optimize the add element to array

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: lambdaInArrayList/LambdaInArraylistProgram.java (hunk 1 above)
Comment (tejashrighogare) [reply lambdaInArrayList/LambdaInArraylistProgram.java]: Code is optimized for add element to array.

File: lambdaInArrayList/LambdaInArraylistProgram.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        System.out.println("After invoking remove(object) method: " + arrayList);
+        arrayList.remove(0);
+        System.out.println("After invoking remove(index) method: " + arrayList);
+        List<String> newListObject = Arrays.asList("Ravi", "Hanumant");
+        ArrayList<String> newArrayList = new ArrayList<String>(newListObject);
+        System.out.println("Updated list : " + newArrayList);
+        arrayList.removeAll(newArrayList);
+        System.out.println("After invoking removeAll() method: " + arrayList);
+        arrayList.removeIf(stringElement -> stringElement.contains("Ajay"));
+        System.out.println("After invoking removeIf() method using lambda function: " + arrayList);
```
Comment (amitsahu8) [inline lambdaInArrayList/LambdaInArraylistProgram.java:21]: add readme for output

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: lambdaInArrayList/LambdaInArraylistProgram.java (hunk 2 above)
Comment (tejashrighogare) [reply lambdaInArrayList/LambdaInArraylistProgram.java:21]: Added ReadMe file

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**